datasets/opensurfaces.yaml

```diff
@@ -0,0 +1,20 @@
+Name: OpenSurfaces
+Description: A large database of annotated surfaces created from real-world consumer photographs.
+Documentation: http://opensurfaces.cs.cornell.edu/publications/opensurfaces/
+Contact: snavely@cs.cornell.edu
+ManagedBy: Cornell University
+UpdateFrequency: Static dataset (not updated)
+Tags:
+  - computer vision
+License: The annotations are licensed under a Creative Commons Attribution 4.0 International License. The photos have their own licenses.
+Resources:
+  - Description: OpenSurfaces data
+    ARN: arn:aws:s3:::labelmaterial
+    Region: us-east-1
+    Type: S3 Bucket
+    Explore: '[Browse data on project webpage](http://opensurfaces.cs.cornell.edu/)'

```

Comment:
This all looks good to me. The only issue I can spot is in this Explore section. Even though there's only one entry, it still needs to be a list to pass validation. A newline and a `-` should fix the validation error. As an example, see https://github.com/awslabs/open-data-registry/blob/main/datasets/noaa-himawari.yaml